packages/venia-concept/src/components/SearchBar/autocomplete.css

```diff
@@ -1,12 +1,57 @@
+.hidden {
+    opacity: 0;
+    transform: translate3d(0, -2rem, 0);
+    transition-duration: 192ms;
```

Comment:
Personally, I don't think so. Unlike timing functions, durations should be custom to each individual element, since they depend on the size of the element and the distance it travels. I do think we've documented the guidelines for them.

Reply:
Excellent point!